src/main/java/bookface/logic/commands/edit/EditUserCommand.java

```diff
@@ -25,27 +26,22 @@
 import bookface.model.tag.Tag;
 
 /**
- * Edits the details of an existing user in the user list.
+ * Deletes a user identified using it's displayed index from the user list.
```

Comment:
The deleted javadoc is correct and it's been replaced with deleteusercommand's javadoc.

Reply:
You're right, fixed!